### PR TITLE
OSDOCS-1952 Document clarification OCP 4.6 bare metal clusters do not support applying network policy to ingress router pods

### PR DESCRIPTION
**Pull request by bmcelvee**
https://issues.redhat.com/browse/OSDOCS-1952

Previews: 
https://deploy-preview-30556--osdocs.netlify.app/openshift-enterprise/latest/networking/network_policy/about-network-policy.html
https://deploy-preview-30556--osdocs.netlify.app/openshift-enterprise/latest/networking/network_policy/multitenant-network-policy.html

### CODE REVIEW

**Comment by mikemckiernan** [comment]
Hi Brandi, I was trolling through PRs (something unrelated) and spotted this PR.  I worked a recent PR on the same subject: https://github.com/openshift/openshift-docs/pull/29633.  The bits are still warm on that one.  Sharing in case that covers what customers need or if you'd like to tweak the language in all places for consistency.

**Comment by bmcelvee** [comment]
@mcurry-rh I'm tinkering with updating the call-outs that @mikemckiernan added recently to add the specifications for bare metal installations. Does this appropriately convey what you said? Thank you!

File: modules/nw-networkpolicy-about.adoc [hunk 1]
```diff
@@ -47,6 +47,8 @@ To make a project allow only connections from the {product-title} Ingress Contro
 [IMPORTANT]
 ====
 For the OVN-Kubernetes network provider plug-in, when the Ingress Controller is configured to use the `HostNetwork` endpoint publishing strategy, there is no supported way to apply network policy so that ingress traffic is allowed and all other traffic is denied.
+
+For bare metal OVN-Kubernetes installs, the Ingress Controller pods default to using the `HostNetwork` endpoint publishing strategy. There is no way to track host-networked pods within OVN. And there is no way to define any kind of network policy, which includes allow all ingress traffic with network policy on bare metal clusters.
```
Comment (jboxman) [inline modules/nw-networkpolicy-about.adoc:51]: @bmcelvee 

I'm referring to this as the _OVN-Kubernetes cluster network provider_ elsewhere.

> If your cluster uses the OVN-Kubernetes cluster network provider, ...

Comment (bmcelvee) [reply modules/nw-networkpolicy-about.adoc:51]: Thanks! I'll update both references.

**Comment by bmcelvee** [comment]
Hi @anuragthehatter, would you mind reviewing this PR please? Thanks!

**Comment by bmcelvee** [comment]
Looks like we do not need this content. I am closing this PR. If anything changes, please let me know.

**Pull request closed without merging**